theme/static/js/hs_file_browser.js

```diff
@@ -940,6 +935,17 @@ function bindFileBrowserItemEvents() {
 
         $(".selection-menu").hide();    // Hide other menus
 
+        // register a listener for clicks outside of the context menu
+        $(document).on('click.menu', function(event) {
```

Comment:
What is `.menu` doing in this event?

Reply:
I'm using the "menu" namespace on the click event
https://api.jquery.com/on/
![image](https://user-images.githubusercontent.com/17934193/156086850-2ee92e9f-f93b-4de0-b968-71a8fbd34243.png)
This is so that we don't de-register ALL click events in $(document)